packages/authentication/index.d.ts

```diff
@@ -1,6 +1 @@
-// Copyright IBM Corp. 2013,2017. All Rights Reserved.
-// Node module: @loopback/authentication
-// This file is licensed under the MIT License.
-// License text available at https://opensource.org/licenses/MIT
-
-export * from './lib';
+export * from './src';
```

Comment:
reverted the changes made unintentionally to authentication, context, and core.  Sorry!